web/packages/teleterm/src/ui/AccessRequests/SelectorMenu.tsx

```diff
@@ -162,6 +174,7 @@ export function SelectorMenu() {
   });
 
   function openMenu(): void {
+    void fetchRequests();
```

Comment:
How did that work before? If I never opened `DocumentAccessRequests`, how would the list in the selector menu get populated?

Reply:
There's a `useEffect` here as well:
```ts
  useEffect(() => {
    if (canUse) {
      void fetchRequests();
    }
  }, [canUse, fetchRequests]);
```